doc/source/train/getting-started-pytorch.rst

```diff
@@ -273,6 +275,14 @@ See :ref:`data-ingest-torch`.
 
     See :ref:`data-ingest-torch` for more details.
 
+.. note::
+    When using PyTorch DataLoader multiprocessing with Ray Train, you should set the
+    ``multiprocessing_context`` to ``multiprocessing.context.ForkServerContext()`` as
+    shown in the example above. This is because Ray Train configures the global logger with
+    Ray-specific context like ``get_actor_id()``, which, when inherited by Torch Dataloader
+    subprocesses using ``fork``, can cause hangs during logging since these subprocesses
+    aren't Ray actors.
```

Comment:
Can you add a reference tag here (ex: `.. _forking-ray-processes-antipattern:`)?

https://github.com/ray-project/ray/blob/master/doc/source/ray-core/patterns/fork-new-processes.rst#L1

Then, just point to this in the note, and no need to go into so much detail about the specific case I ran into. 

```suggestion
.. note::
    When using PyTorch DataLoader multiprocessing with Ray Train, you should set the process start method to be `forkserver` or `spawn` as
    shown in the example above. :ref:`Forking Ray Actors and Tasks is an anti-pattern <forking-ray-processes-antipattern>` that
    can lead to unexpected issues such as deadlocks.
```